app/src/main/java/ch/epfl/sdp/blindly/recording/RecordingActivity.kt

```diff
@@ -105,6 +132,13 @@ class RecordingActivity : AppCompatActivity(), AudioLibraryAdapter.OnItemClickLi
             setOutputFormat(MediaRecorder.OutputFormat.THREE_GPP)
             setAudioEncoder(MediaRecorder.AudioEncoder.AMR_WB)
             setOutputFile(recordFilePath)
+            setMaxDuration(MAXIMUM_AUDIO_DURATION)
+            setOnInfoListener { mr, what, extra ->
```

Comment:
If you don't use mr ant extra I think you can put _ instead of naming the values

Reply:
True !